recipe/meta.yaml

```diff
@@ -39,9 +39,9 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.37.*
+    - llvmlite 0.38.*
     - numpy
-    - tbb-devel >=2021.0
+    - tbb-devel 2021.*
```

Comment:
@stuartarchibald wdyt? IOW just pin during the build, but relax during runtime. This won't require a hot-fix. If we do find it has issues with 2022, we can pin here and hot-fix after

Reply:
Isn't that what I did in the most recent comment? tbb-devel is capped during the build, but tbb is not capped at runtime (and already excluded from exporting)

(Just checking to see if the current recipe state is the expected one)

Reply:
Yep it is. Just wanted to check with Stuart

Reply:
Seems like that could work WRT the comments in https://github.com/numba/numba/pull/7608. I think that given there's a chance to fix the repo metadata later to deal with any problems discovered, this seems less of a concern in general. I'm glad this as been discussed here such that for 0.56 there's something to refer back to. Thanks for your input!